python/tskit/provenance.py

```diff
@@ -132,3 +135,44 @@ def validate_provenance(provenance):
         jsonschema.validate(provenance, schema)
     except jsonschema.exceptions.ValidationError as ve:
         raise exceptions.ProvenanceValidationError from ve
+
+
+def provenance_command_JSON(exclude_args=[]):
```

Comment:
You don't want to have a list as a function parameter default: [surprising things happen](https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments). ``None`` is safer.

Reply:
Ah - I didn't know that. Thanks for the tip.

Reply:
Done